### PR TITLE
Add missing word

### PR DESCRIPTION
**Pull request by plumbis**
Signed-off-by: Pete Lumbis <pete@upbound.io>

### Description of your changes
Adds the word "the" that was missing. This was caught in [docs site PR #208](https://github.com/crossplane/crossplane.github.io/pull/208)

Fixes #

I have:

- [ ] Read and followed Crossplane's [contribution process].
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

**Comment by plumbis** [comment]
The Docs Validation failure is because of https://github.com/crossplane/crossplane.github.io/issues/214

File: docs/guides/vault-as-secret-store.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ which expect input parameters that could be sensitive. Initial password of a
 managed database is a good example of this category.
 
 It is already possible to use Vault for the 1st category (i.e. Provider
-Credentials) as described in [the previous guide]. 3rd use case is a relatively
+Credentials) as described in [the previous guide]. The 3rd use case is a relatively
```
Comment (hasheddan) [inline docs/guides/vault-as-secret-store.md:29]:
```suggestion
Credentials) as described in [the previous guide]. The 3rd use case is relatively
```

**Review by hasheddan — COMMENTED** [review]
@plumbis can you make the change below across all branches?

File: docs/guides/vault-as-secret-store.md (hunk 1 above)
Comment (plumbis) [reply docs/guides/vault-as-secret-store.md]: done.

**Review by hasheddan — APPROVED** [review]

**Pull request merged**